### DISCUSSION TITLE
Handling duplicate nodules after detection

### DISCUSSION THREAD

**Discussion by DSRajesh** [Project-MONAI/MONAI#7453] [Q&A]
Hello

How are duplicate nodules handled during nodule detection so that no nodule is reported more than once, in MONAI lung CT nodule detection ?

Thanks
Rajesh D S

**Comment by kretes** [comment]
In detection you usually apply a variant of non-maximum suppression. In monai there is this implementation: https://docs.monai.io/en/latest/data.html#monai.data.box_utils.batched_nms
